config.json

```diff
@@ -2541,8 +2533,8 @@
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-measurement-base",
-        "version": "21.1.0",
-        "nugetVersion": "121.1.0",
+        "version": "16.3.0",
```

Comment:
This should be examined, it's a large downgrade.

Reply:
Good spot. 
I had to tune some package versions. OK. I will check this one.

Reply:
removal of:
```
      {
        "groupId": "com.google.android.gms",
        "artifactId": "play-services-measurement-base",
        "version": "16.3.0",
        "nugetVersion": "116.3.0",
        "nugetId": "Xamarin.GooglePlayServices.Measurement.Base",
        "dependencyOnly": true
      },
```
causes:
```
No matching artifact config found for: 
			com.google.android.gms.play-services-measurement-base:[16.3.0]
to satisfy dependency of: 
			com.google.firebase.firebase-analytics-impl:16.3.0
```